src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="search"
+      data-qa="big">
+      <label>
+        <input
+        class="bar_b bars"
+        data-qa="keypress"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        >
```

Comment:
Pay attention on indents
```suggestion
        <input
          class="bar_b bars"
          data-qa="keypress"
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```